scripts/split-e2e-tests.ts

```diff
@@ -71,7 +71,11 @@ type CandidateJob = {
   runSolo: boolean;
 };
 const createJob = (os: OS_TYPE, jobIdx: number, runSolo: boolean = false): CandidateJob => {
-  const region = AWS_REGIONS_TO_RUN_TESTS[jobIdx % AWS_REGIONS_TO_RUN_TESTS.length];
+  // The bash terminal for Windows on CodeBuild is non-interactive.
+  // amplify-configure will always choose us-east-1 for the region.
+  // This will set all Windows jobs to use us-east-1 to avoid region mismatch.
+  // We will come back to this later to properly fix the testing issue.
+  const region = os === 'w' ? 'us-east-1' : AWS_REGIONS_TO_RUN_TESTS[jobIdx % AWS_REGIONS_TO_RUN_TESTS.length];
```

Comment:
Another weird thing, this only seems to affect the `amplify configure` step. On command like `codegen add` the selection is set correctly.

Reply:
We might need to look into how we're sending the region selection command interactively and make it for both platforms. Running in `us-east-1` is fine for now.

Reply:
It is using j/k to navigate the list. I tried with up/down arrow as well and ran into the same issue. I've created a task to investigate further.